service/collector/bridge.go

```diff
@@ -91,11 +91,13 @@ func (c *Bridge) Collect(ch chan<- prometheus.Metric) error {
 		}
 
 		for _, i := range interfaces {
-			id := clusterIDFromName(i.Name)
-			if id == "" {
+			id, err := clusterIDFromName(i.Name)
+			if IsExecutionFailed(err) {
```

Comment:
It feels weird to handle `executionFailedError`. I think it can bite us in the future. My suggestion would be `nameMatchError`. But anything else than `executionFailedError` works.

Reply:
Can you elaborate on your idea about this specific error type? I think we are not on the same page. Why should this error not be handled purposefully?

Reply:
We discussed and changed the name. `fmt` update will follow.

Reply:
Here https://github.com/giantswarm/fmt/pull/57.